priv/repo/migrations/20231207024956_add_channels_table.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Slax.Repo.Migrations.AddChannelsTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:channels) do
+      add(:channel_id, :string)
```

Comment:
I think we probably want a unique index on this field so we don't end up with multiple copies (and so it's faster for the DB to locate a particular channel)

Reply:
We might want a regular index on the `disabled` field, too, for efficient filtering